### PR TITLE
[opentelemetry-operator] read cert contents

### PR DESCRIPTION
**Pull request by jinja2**
[Issue Link](https://github.com/open-telemetry/opentelemetry-helm-charts/issues/1120)

The operator chart has been updated to read contents of the file path provided in below variables.
```
  ## TLS Certificate Option 3: Use your own self-signed certificate.
  ## certManager and autoGenerateCert must be disabled and cert_file, key_file, and ca_file must be set.
  ## Path to your own PEM-encoded certificate.
  cert_file: ""
  ## Path to your own PEM-encoded private key.
  key_file: ""
  ## Path to the CA cert.
  ca_file: ""
```

Screenshot to verify that the contents are read and added to secret correctly.

Screenshot 2024-04-05 at 1 44 11 PM

### CODE REVIEW

File: charts/opentelemetry-operator/templates/_helpers.tpl [hunk 1]
```diff
@@ -117,9 +117,9 @@ a cert is loaded from an existing secret or is provided via `.Values`
 {{- $caCertEnc = b64enc $ca.Cert }}
 {{- end }}
 {{- else }}
-{{- $certCrtEnc = b64enc .Values.admissionWebhooks.cert_file }}
-{{- $certKeyEnc = b64enc .Values.admissionWebhooks.key_file }}
-{{- $caCertEnc = b64enc .Values.admissionWebhooks.ca_file }}
+{{- $certCrtEnc = .Files.Get .Values.admissionWebhooks.cert_file | b64enc }}
```
Comment (jinja2) [inline charts/opentelemetry-operator/templates/_helpers.tpl:120]: I would prefer to rename these variables to be camelCase for consistency. Let me know if sounds okay.

Comment (TylerHelmuth) [reply charts/opentelemetry-operator/templates/_helpers.tpl:120]: Yup

**Review by jvoravong — APPROVED** [review]

**Review by Allex1 — APPROVED** [review]

File: charts/opentelemetry-operator/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.53.2
+version: 0.53.3
```
Comment (TylerHelmuth) [inline charts/opentelemetry-operator/Chart.yaml:3]: Since this is a breaking change lets make this a 0.54.0

Comment (jinja2) [reply charts/opentelemetry-operator/Chart.yaml]: done!

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**